control-plane/pkg/reconciler/trigger/trigger.go

```diff
@@ -87,6 +87,7 @@ type Reconciler struct {
 }
 
 func (r *Reconciler) ReconcileKind(ctx context.Context, trigger *eventing.Trigger) reconciler.Event {
+	trigger.Status.MarkOIDCIdentityCreatedNotSupported()
```

Comment:
Do we have an issue tracking this?

Reply:
I just made https://github.com/knative-extensions/eventing-kafka-broker/issues/3378